From: KevinDuan <duankaiwen@kylinos.com>
Date: Tue, 21 May 2024 03:00:53 +0000
Subject: Translated using Weblate (Uyghur)

Currently translated at 100.0% (153 of 153 strings)

Translation: openkylin-nile-new/peony-send-to-connectivity
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/peony-send-to-connectivity/ug/
---
 translations/peony-send-to-connectivity_ug.ts | 306 +++++++++++++-------------
 1 file changed, 153 insertions(+), 153 deletions(-)

diff --git a/translations/peony-send-to-connectivity_ug.ts b/translations/peony-send-to-connectivity_ug.ts
index 672f1fd..95bf486 100644
--- a/translations/peony-send-to-connectivity_ug.ts
+++ b/translations/peony-send-to-connectivity_ug.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../ui/filemanageview/androidhomepage.cpp" line="14"/>
         <source>Storage</source>
-        <translation type="unfinished"></translation>
+        <translation>ساقلاش</translation>
     </message>
 </context>
 <context>
@@ -14,47 +14,47 @@
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="69"/>
         <source>Picture</source>
-        <translation type="unfinished"></translation>
+        <translation>رەسىم</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="72"/>
         <source>Video</source>
-        <translation type="unfinished"></translation>
+        <translation>ۋىدېئو</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="75"/>
         <source>Music</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا-مۇزىكا</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="78"/>
         <source>Document</source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="81"/>
         <source>WeChat</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۈندىدار</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="84"/>
         <source>QQ</source>
-        <translation type="unfinished"></translation>
+        <translation>QQ</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="87"/>
         <source>Storage</source>
-        <translation type="unfinished"></translation>
+        <translation>ساقلاش</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="100"/>
         <source>All File</source>
-        <translation type="unfinished"></translation>
+        <translation>بارلىق ھۆججەت</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="102"/>
         <source>items</source>
-        <translation type="unfinished"></translation>
+        <translation>تۈر</translation>
     </message>
 </context>
 <context>
@@ -62,12 +62,12 @@
     <message>
         <location filename="../projection/device/basedevice.cpp" line="404"/>
         <source>Control Devices Supported</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۈسكۈنىنى تەتۈر كونترول قىلىشنى قوللايدۇ</translation>
     </message>
     <message>
         <location filename="../projection/device/basedevice.cpp" line="408"/>
         <source>Control device not supported</source>
-        <translation type="unfinished"></translation>
+        <translation>ھازىرقى ئۇلىنىش ئۈسكىنىنى كونترول قىلىشنى قوللىمايدۇ</translation>
     </message>
 </context>
 <context>
@@ -75,22 +75,22 @@
     <message>
         <location filename="../ui/connectinterface/connectinterface.cpp" line="19"/>
         <source>Back</source>
-        <translation type="unfinished"></translation>
+        <translation>كەينىگە يېنىش</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/connectinterface.cpp" line="33"/>
         <source>Nearby device</source>
-        <translation type="unfinished"></translation>
+        <translation>يېقىن ئەتراپتىكى ئۈسكۈنە</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/connectinterface.cpp" line="34"/>
         <source>Connect Phone</source>
-        <translation type="unfinished"></translation>
+        <translation>تېلېفوننى ئۇلاش</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/connectinterface.cpp" line="35"/>
         <source>Device Code</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۈسكۈنە كودى ئارقىلىق ئۇلاش</translation>
     </message>
 </context>
 <context>
@@ -98,24 +98,24 @@
     <message>
         <location filename="../ui/view/connectedwin.cpp" line="38"/>
         <source>CONNECTED</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۇلاندى</translation>
     </message>
     <message>
         <location filename="../ui/view/connectedwin.cpp" line="114"/>
         <location filename="../ui/view/connectedwin.cpp" line="122"/>
         <source>ComputerScreen</source>
-        <translation type="unfinished"></translation>
+        <translation>كومپيۇتېر ئېكرانىنى پىرويېكسىيەلەش</translation>
     </message>
     <message>
         <location filename="../ui/view/connectedwin.cpp" line="120"/>
         <source>MobileScreen</source>
-        <translation type="unfinished"></translation>
+        <translation>يانفون ئېكرانىنى پىرويېكسىيەلەش</translation>
     </message>
     <message>
         <location filename="../ui/view/connectedwin.cpp" line="138"/>
         <location filename="../ui/view/connectedwin.cpp" line="182"/>
         <source>ExitScreen</source>
-        <translation type="unfinished"></translation>
+        <translation>پىرويېكسىيەلەشتىن چېكىنىش</translation>
     </message>
 </context>
 <context>
@@ -123,17 +123,17 @@
     <message>
         <location filename="../ui/connectinterface/devicecodewidget.cpp" line="31"/>
         <source>Please enter the connection code of the other device</source>
-        <translation type="unfinished"></translation>
+        <translation>قارشى تەرەپ ئۈسكۈنىسىنىڭ ئۇلىنىش كودىنى كىرگۈزۈڭ</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/devicecodewidget.cpp" line="40"/>
         <source>The connection code of the device can be obtained on the homepage of the other party&apos;s &apos;multi terminal collaboration&apos; application</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۈسكۈنىنىڭ ئۇلىنىش كودىغا قارشى تەرەپنىڭ «كۆپ تېرمىنال ھەمكارلىقى» پروگراممىسىنىڭ باش بېتىدىن ئېرىشكىلى بولىدۇ.</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/devicecodewidget.cpp" line="69"/>
         <source>Connection</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۇلاش</translation>
     </message>
 </context>
 <context>
@@ -141,40 +141,40 @@
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="25"/>
         <source>Go Back</source>
-        <translation type="unfinished"></translation>
+        <translation>قايت</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="37"/>
         <source>Go Forward</source>
-        <translation type="unfinished"></translation>
+        <translation>ئالدىغا ماڭ</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="57"/>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="406"/>
         <source>Search File</source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت ئىزدەش</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="85"/>
         <source>Select</source>
-        <translation type="unfinished"></translation>
+        <translation>تاللاش</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="106"/>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="626"/>
         <source>List Mode</source>
-        <translation type="unfinished"></translation>
+        <translation>تىزىملىك ھالىتى</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="118"/>
         <source>Refresh</source>
-        <translation type="unfinished"></translation>
+        <translation>يېڭىلاش</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="124"/>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="432"/>
         <source>Select File</source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت تاللاش</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="133"/>
@@ -183,12 +183,12 @@
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="650"/>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="658"/>
         <source>Select All</source>
-        <translation type="unfinished"></translation>
+        <translation>ھەممىنى تاللاڭ</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="142"/>
         <source>Finish</source>
-        <translation type="unfinished"></translation>
+        <translation>تاماملاش</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="210"/>
@@ -196,52 +196,52 @@
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="649"/>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="657"/>
         <source>Deselect All</source>
-        <translation type="unfinished"></translation>
+        <translation>تاللاشنى بىكار قىلىش</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.cpp" line="631"/>
         <source>Icon Mode</source>
-        <translation type="unfinished"></translation>
+        <translation>سىنبەلگە ھالىتى</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.h" line="91"/>
         <source>List of Files</source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت تىزىملىكى</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.h" line="92"/>
         <source>Picture</source>
-        <translation type="unfinished"></translation>
+        <translation>رەسىم</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.h" line="93"/>
         <source>Video</source>
-        <translation type="unfinished"></translation>
+        <translation>ۋىدېئو</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.h" line="94"/>
         <source>Music</source>
-        <translation type="unfinished"></translation>
+        <translation>ناخشا-مۇزىكا</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.h" line="95"/>
         <source>Doc</source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.h" line="96"/>
         <source>QQ</source>
-        <translation type="unfinished"></translation>
+        <translation>QQ</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.h" line="97"/>
         <source>WeChat</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۈندىدار</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/filemanagewin.h" line="98"/>
         <source>Storage</source>
-        <translation type="unfinished"></translation>
+        <translation>ساقلاش</translation>
     </message>
 </context>
 <context>
@@ -250,17 +250,17 @@
         <location filename="../ui/filemanageview/fileview.cpp" line="132"/>
         <location filename="../ui/filemanageview/fileview.cpp" line="525"/>
         <source>Download</source>
-        <translation type="unfinished"></translation>
+        <translation>كومپيۇتېرغا چۈشۈرۈش</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/fileview.cpp" line="405"/>
         <source>Choose folder</source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت قىسقۇچنى تاللاش</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/fileview.cpp" line="523"/>
         <source>Open</source>
-        <translation type="unfinished"></translation>
+        <translation>ئېچىش</translation>
     </message>
 </context>
 <context>
@@ -268,42 +268,42 @@
     <message>
         <location filename="../ui/view/homepage.cpp" line="42"/>
         <source>Multiterminal</source>
-        <translation type="unfinished"></translation>
+        <translation>كۆپ تېرمىنال ھەمكارلىقى</translation>
     </message>
     <message>
         <location filename="../ui/view/homepage.cpp" line="48"/>
         <source>Cross equipment and cross system collaboration. It is more convenient to share resources and screens, and more convenient and efficient to work!</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۈسكۈنە، سەھنە ھەمكارلىقى، ھۆججەت، ئېكراننى ئەمەلىي ھەمبەھرىلەشتىن ھالقىپ ئۆتۈپ، يۇقىرى ئۈنۈملۈك خىزمەت قىلىش ئۇسۇلىنى بارلىققا كەلتۈردى!</translation>
     </message>
     <message>
         <location filename="../ui/view/homepage.cpp" line="55"/>
         <source>Device Code:</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۈسكۈنە كودى:</translation>
     </message>
     <message>
         <location filename="../ui/view/homepage.cpp" line="66"/>
         <source>Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت قىسقۇچ</translation>
     </message>
     <message>
         <location filename="../ui/view/homepage.cpp" line="70"/>
         <source>Share</source>
-        <translation type="unfinished"></translation>
+        <translation>ئېكراننى ھەمبەھرىلەش</translation>
     </message>
     <message>
         <location filename="../ui/view/homepage.cpp" line="74"/>
         <source>Cross</source>
-        <translation type="unfinished"></translation>
+        <translation>ئېكران ھالقىپ كونترول قىلىش</translation>
     </message>
     <message>
         <location filename="../ui/view/homepage.cpp" line="77"/>
         <source>Connect Now</source>
-        <translation type="unfinished"></translation>
+        <translation>دەرھال ئۇلاش</translation>
     </message>
     <message>
         <location filename="../ui/view/homepage.cpp" line="185"/>
         <source>No network</source>
-        <translation type="unfinished"></translation>
+        <translation>تورغا ئۇلانمىدى</translation>
     </message>
 </context>
 <context>
@@ -312,97 +312,97 @@
         <location filename="../ui/mainwindow.cpp" line="379"/>
         <location filename="../ui/mainwindow.cpp" line="1376"/>
         <source>kylin-connectivity</source>
-        <translation type="unfinished"></translation>
+        <translation>كۆپ تېرمىنالنى بىرلەشتۈرۈش</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="479"/>
         <source>Agreed to connect</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۇلىنىشقا قوشۇلۇش</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="481"/>
         <source>Peer has agreed</source>
-        <translation type="unfinished"></translation>
+        <translation>قارشى تەرەپ قوشۇلدى</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="483"/>
         <source>Establishing connection, please wait...</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۇلىنىۋاتىدۇ، سەل ساقلاڭ...</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="487"/>
         <source>CANCEL</source>
-        <translation type="unfinished"></translation>
+        <translation>بىكار قىلىش</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="935"/>
         <source>Not currently connected, please connect</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۇلانمىغان ھالەتتە، ئۇلاڭ</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="693"/>
         <location filename="../ui/mainwindow.cpp" line="1364"/>
         <source>file download failed</source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت چۈشۈرۈش مەغلۇپ بولدى</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1082"/>
         <source>Connection request received from&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>-دىن كەلگەن &quot;نى تاپشۇرۇۋالدى</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1082"/>
         <location filename="../ui/mainwindow.cpp" line="1404"/>
         <source>&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>&quot;ئىلتىماس</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1083"/>
         <source>After consent, the other party can view and download all the files on the device, and can share the other party&apos;s desktop to this screen.</source>
-        <translation type="unfinished"></translation>
+        <translation>قوشۇلغاندىن كېيىن قارشى تەرەپ بۇ ئۈسكۈنىدىكى بارلىق ھۆججەتلەرنى كۆرەلەيدۇ ۋە چۈشۈرەلەيدۇ، قارشى تەرەپنىڭ ئېكرانىنى مۇشۇ ئېكرانغا ھەمبەھرىلىيەلەيدۇ.</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1089"/>
         <location filename="../ui/mainwindow.cpp" line="1410"/>
         <source>NO</source>
-        <translation type="unfinished"></translation>
+        <translation>رەت قىلىش</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1088"/>
         <location filename="../ui/mainwindow.cpp" line="1409"/>
         <source>YES</source>
-        <translation type="unfinished"></translation>
+        <translation>قوشۇلۇش</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1346"/>
         <source>File open exception!</source>
-        <translation type="unfinished"></translation>
+        <translation>ھۆججەت ئېچىش بىنورمال!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1423"/>
         <source>The other party has refused your screen projection request!</source>
-        <translation type="unfinished"></translation>
+        <translation>قارشى تەرەپ سىزنىڭ ئىكراننى پىرويېكسىيەلەش تەلىپىڭىزنى رەت قىلدى!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1424"/>
         <source>Failed to cast the screen. Please contact the other party and try again.</source>
-        <translation type="unfinished"></translation>
+        <translation>ئېكران پىرويېكسىيەلەش مەغلۇپ بولدى. قارشى تەرەپ بىلەن ئالاقىلىشىپ قايتا سىناڭ.</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1108"/>
         <location filename="../ui/mainwindow.cpp" line="1428"/>
         <source>RECONNECT</source>
-        <translation type="unfinished"></translation>
+        <translation>قايتا ئۇلاش</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1103"/>
         <source>The other party has rejected your connection request!</source>
-        <translation type="unfinished"></translation>
+        <translation>قارشى تەرەپ سىزنىڭ ئۇلاش تەلىپىڭىزنى رەت قىلدى!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1104"/>
         <source>Connection failed. Please contact the other party and try again.</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۇلاش مەغلۇپ بولدى. قارشى تەرەپ بىلەن ئالاقىلىشىپ قايتا سىناڭ.</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1109"/>
@@ -411,120 +411,120 @@
         <location filename="../ui/mainwindow.cpp" line="1803"/>
         <location filename="../ui/mainwindow.cpp" line="1821"/>
         <source>CLOSE</source>
-        <translation type="unfinished"></translation>
+        <translation>ئېتىش</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1207"/>
         <source>Please install kylin-assistant on the Android terminal!</source>
-        <translation type="unfinished"></translation>
+        <translation>يانفون نۇسخىسىغا چىلىن يانفون ياردەمچىسى ئەپىنى چۈشۈڭ!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1217"/>
         <source>Please use the USB to connect your phone device!</source>
-        <translation type="unfinished"></translation>
+        <translation>يانفونىڭىزنى USB ئارقىلىق ئۇلاڭ</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1228"/>
         <source>Connection error</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۇلاش خاتالىقى</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1230"/>
         <source>Connection timed out</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۇلىنىش ۋاقتى ئاشتى</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="2006"/>
         <source>There is currently a connection in progress!</source>
-        <translation type="unfinished"></translation>
+        <translation>نۆۋەتتە ئۇلىنىش داۋاملىشىۋاتىدۇ!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1377"/>
         <source>Version:</source>
-        <translation type="unfinished"></translation>
+        <translation>نەشرى:</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1259"/>
         <source>No Content</source>
-        <translation type="unfinished"></translation>
+        <translation>مەزمۇن يوق</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1379"/>
         <source>Mobile Assistant is an interconnection tool of Android device and Kirin operating system, which supports Android file synchronization, file transfer, screen mirroring and other functions, which is simple and fast to operate</source>
-        <translation type="unfinished"></translation>
+        <translation>يانفون ياردەمچىسى ئاندىرويىد ئۈسكۈنىسى بىلەن Kirin مەشغۇلات سىستېمىسىنىڭ ئۆزئارا تۇتاشتۇرۇش قورالى بولۇپ، ئاندىرويىد ھۆججەتلىرىنى ماسلاشتۇرۇش، ھۆججەت يۆتكەش، ئېكران ئەينىكى قاتارلىق ئىقتىدارلارنى قوللايدۇ، مەشغۇلاتى ئاددىي ھەم تېز</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1403"/>
         <source>Received screen projection request from &quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>تاپشۇرۇۋالغان ئېكران پروزىلاش ئىلتىماسى &quot;</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1405"/>
         <source>After consent, the other party can share the device desktop to this screen.</source>
-        <translation type="unfinished"></translation>
+        <translation>ماقۇل بولغاندىن كېيىن قارشى تەرەپ ئۈسكۈنە ئۈستەل يۈزىنى مۇشۇ ئېكرانغا ھەمبەھىرلىسە بولىدۇ.</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1449"/>
         <source>The other party agreed to your screen projection request!</source>
-        <translation type="unfinished"></translation>
+        <translation>قارشى تەرەپ سىزنىڭ ئىكراندىكى پرويېكسىيە تەلىپىڭىزگە قوشۇلدى!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1450"/>
         <source>The screen is being cast, please wait...</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىكران قۇيۇلىۋاتىدۇ، ساقلاپ تۇرۇڭ...</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1475"/>
         <location filename="../ui/mainwindow.cpp" line="1498"/>
         <source>End of screen projection</source>
-        <translation type="unfinished"></translation>
+        <translation>ئېكران پروزىلاشنىڭ ئاخىرلىشىشى</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1476"/>
         <location filename="../ui/mainwindow.cpp" line="1499"/>
         <source>The other party has finished the screen projection function.</source>
-        <translation type="unfinished"></translation>
+        <translation>قارشى تەرەپ ئېكران پروزىلاش ئىقتىدارىنى تاماملاپ بولدى.</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1572"/>
         <source>Search data loading failed!</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىزدەش سانلىق مەلۇماتى قاچىلاش مەغلۇپ بولدى!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1590"/>
         <location filename="../ui/mainwindow.cpp" line="1593"/>
         <source>Transmission interruption</source>
-        <translation type="unfinished"></translation>
+        <translation>يوللاش ئۈزۈلۈپ قېلىش</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1591"/>
         <source>The other party&apos;s device has insufficient local storage!</source>
-        <translation type="unfinished"></translation>
+        <translation>قارشى تەرەپنىڭ ئۈسكۈنىسى يەرلىك ساقلاش يېتەرلىك ئەمەس!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1593"/>
         <source>Insufficient local storage space!</source>
-        <translation type="unfinished"></translation>
+        <translation>يەرلىك ساقلاش بوشلۇقى يېتەرلىك ئەمەس!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1612"/>
         <source>No relevant results</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇناسىۋەتلىك نەتىجە يوق</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1815"/>
         <source>Not currently supported</source>
-        <translation type="unfinished"></translation>
+        <translation>ھازىر قوللىمايمەن</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1817"/>
         <source>The Wayland version system currently does not support mobile screen projection function.</source>
-        <translation type="unfinished"></translation>
+        <translation>Wayland نۇسخىسى سىستېمىسى ۋاقتىنچە يانفوننىڭ ئېكرانغا تاشلاش ئىقتىدارىنى قوللىمايدۇ</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="2020"/>
         <source>The screen projection request has been sent to the connected device. Please click [Agree] in the opposite pop-up window</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۇلانغان ئۈسكۈنىگە ئېكران پروزىلاش تەلىپى ئەۋەتىلدى. قارشى تەرەپتىكى چوڭ-چوڭ كۆزنەكتە [Agree] نى چېكىڭ</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="937"/>
@@ -541,40 +541,40 @@
         <location filename="../ui/mainwindow.cpp" line="2008"/>
         <location filename="../ui/mainwindow.cpp" line="2025"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>ماقۇل</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1485"/>
         <location filename="../ui/mainwindow.cpp" line="1797"/>
         <source>Screen projection loading error</source>
-        <translation type="unfinished"></translation>
+        <translation>ئېكران پروزىسى قاچىلاش خاتالىقى</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1487"/>
         <location filename="../ui/mainwindow.cpp" line="1799"/>
         <source>Please check whether to install the projection expansion package [kylin connectivity tools]</source>
-        <translation type="unfinished"></translation>
+        <translation>پرويېكسىيە كېڭەيتىش بولىقى [Kylin ئۇلىنىش قورالى] نى قاچىلاش ياكى قاچىلىماسلىقنى تەكشۈرۈڭ</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1787"/>
         <source>Uploaded to</source>
-        <translation type="unfinished"></translation>
+        <translation>يوللانغان ۋاقتى:</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1758"/>
         <source>Downloaded to</source>
-        <translation type="unfinished"></translation>
+        <translation>چۈشۈرۈلدى</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1996"/>
         <location filename="../ui/mainwindow.cpp" line="2018"/>
         <source>Request sent successfully!</source>
-        <translation type="unfinished"></translation>
+        <translation>مۇۋەپپىقىيەتلىك ھالدا ئەۋەتىلگەن تەلەپ!</translation>
     </message>
     <message>
         <location filename="../ui/mainwindow.cpp" line="1997"/>
         <source>The connection request has been sent to the selected device. Please click [YES] in the opposite pop-up window</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۇلىنىش ئىلتىماسى تاللانغان ئۈسكۈنىگە ئەۋەتىلدى. قارشى تەرەپتىكى چوڭ-چوڭ كۆزنەكتە [YES] نى چېكىڭ</translation>
     </message>
 </context>
 <context>
@@ -582,12 +582,12 @@
     <message>
         <location filename="../ui/basewidget/messagedialog.cpp" line="29"/>
         <source>kylin-connectivity</source>
-        <translation type="unfinished"></translation>
+        <translation>كۆپ تېرمىنالنى بىرلەشتۈرۈش</translation>
     </message>
     <message>
         <location filename="../ui/basewidget/messagedialog.cpp" line="40"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>تاقاش</translation>
     </message>
 </context>
 <context>
@@ -595,38 +595,38 @@
     <message>
         <location filename="../ui/connectinterface/mobileconnectwin.cpp" line="24"/>
         <source>ScanCode</source>
-        <translation type="unfinished"></translation>
+        <translation>سىكاننېرلاش ئارقىلىق ئۇلاش</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/mobileconnectwin.cpp" line="25"/>
         <source>USBConnect</source>
-        <translation type="unfinished"></translation>
+        <translation>USB ئارقىلىق ئۇلاش</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/mobileconnectwin.cpp" line="43"/>
         <location filename="../ui/connectinterface/mobileconnectwin.cpp" line="103"/>
         <source>Your phone model: </source>
-        <translation type="unfinished"></translation>
+        <translation>يانفونىڭىزنىڭ تىپى: </translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/mobileconnectwin.h" line="49"/>
         <source>vivo</source>
-        <translation type="unfinished"></translation>
+        <translation>ۋىۋو</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/mobileconnectwin.h" line="50"/>
         <source>HUAWEI</source>
-        <translation type="unfinished"></translation>
+        <translation>خۇاۋېي</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/mobileconnectwin.h" line="51"/>
         <source>Xiaomi</source>
-        <translation type="unfinished"></translation>
+        <translation>شىياۋمى</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/mobileconnectwin.h" line="53"/>
         <source>OPPO</source>
-        <translation type="unfinished"></translation>
+        <translation>ئوپپو</translation>
     </message>
 </context>
 <context>
@@ -634,17 +634,17 @@
     <message>
         <location filename="../ui/connectinterface/mobileqrcode.cpp" line="27"/>
         <source>No network</source>
-        <translation type="unfinished"></translation>
+        <translation>تورغا ئۇلانمىدى</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/mobileqrcode.cpp" line="83"/>
         <source>Connect the mobile phone and computer to the same network,open the mobile phone app and scan the QR code.</source>
-        <translation type="unfinished"></translation>
+        <translation>يانفون ۋە كومپيۇتېرنى ئوخشاش تورغا ئۇلاپ، يانفون ئەپ دېتالىنى ئېچىپ، بۇ ئىككىلىك كودنى سىكاننېرلاڭ.</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/mobileqrcode.cpp" line="86"/>
         <source>Use the mobile app to scan this code</source>
-        <translation type="unfinished"></translation>
+        <translation>يانفۇن ئەپ دېتالى ئارقىلىق بۇ كودنى سىكاننېرلاش</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/mobileqrcode.cpp" line="45"/>
@@ -652,7 +652,7 @@
         <location filename="../ui/connectinterface/mobileqrcode.cpp" line="95"/>
         <location filename="../ui/connectinterface/mobileqrcode.cpp" line="101"/>
         <source>No app installed? Install Now</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەپ قاچىلانمىغان؟ دەرھال قاچىلاش</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/mobileqrcode.cpp" line="50"/>
@@ -660,12 +660,12 @@
         <location filename="../ui/connectinterface/mobileqrcode.cpp" line="172"/>
         <location filename="../ui/connectinterface/mobileqrcode.cpp" line="177"/>
         <source>view supported phone types&gt;&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>قوللايدىغان تېلېفون تىپلىرىنى كۆرۈش»</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/mobileqrcode.cpp" line="182"/>
         <source>Please scan this QR code with your mobile browser to download the app</source>
-        <translation type="unfinished"></translation>
+        <translation>بۇ ئىككىلىك كودنى يانفون تور كۆرگۈچى ئارقىلىق سىكاننېرلاپ ئەپنى چۈشۈرۈڭ</translation>
     </message>
 </context>
 <context>
@@ -673,27 +673,27 @@
     <message>
         <location filename="../projection/uibase/scrollsettingwidget.cpp" line="22"/>
         <source>Mouse sensitivity</source>
-        <translation type="unfinished"></translation>
+        <translation>مائۇسنىڭ سەزگۈرلۈكى</translation>
     </message>
     <message>
         <location filename="../projection/uibase/scrollsettingwidget.cpp" line="34"/>
         <source>slow</source>
-        <translation type="unfinished"></translation>
+        <translation>ئاستا</translation>
     </message>
     <message>
         <location filename="../projection/uibase/scrollsettingwidget.cpp" line="35"/>
         <source>quick</source>
-        <translation type="unfinished"></translation>
+        <translation>تېز</translation>
     </message>
     <message>
         <location filename="../projection/uibase/scrollsettingwidget.cpp" line="40"/>
         <source>ok</source>
-        <translation type="unfinished"></translation>
+        <translation>جەزملەشتۈرۈش</translation>
     </message>
     <message>
         <location filename="../projection/uibase/scrollsettingwidget.cpp" line="41"/>
         <source>cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>بىكار قىلىش</translation>
     </message>
 </context>
 <context>
@@ -701,17 +701,17 @@
     <message>
         <location filename="../ui/connectinterface/searchdevicewin.cpp" line="40"/>
         <source>Searching for nearby available devices...</source>
-        <translation type="unfinished"></translation>
+        <translation>يېقىن ئەتراپتىكى ئىشلەتكىلى بولىدىغان ئۈسكۈنىلەرنى ئىزدەۋاتىدۇ...</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/searchdevicewin.cpp" line="77"/>
         <source>No network</source>
-        <translation type="unfinished"></translation>
+        <translation>تورغا ئۇلانمىدى</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/searchdevicewin.cpp" line="179"/>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىزدە</translation>
     </message>
 </context>
 <context>
@@ -720,28 +720,28 @@
         <location filename="../pcscreenmanage/suspendtabbar.cpp" line="101"/>
         <location filename="../pcscreenmanage/suspendtabbar.cpp" line="176"/>
         <source>Back to the desktop</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۈستەل يۈزىگە قايتىش</translation>
     </message>
     <message>
         <location filename="../pcscreenmanage/suspendtabbar.cpp" line="105"/>
         <source>Resume screen projection</source>
-        <translation type="unfinished"></translation>
+        <translation>ئېكران پىرويېكسىيەلەشنى ئەسلىگە قايتۇرۇش</translation>
     </message>
     <message>
         <location filename="../pcscreenmanage/suspendtabbar.cpp" line="175"/>
         <source>Counter control (activate Ctrl+H, cancel Ctrl+G)</source>
-        <translation type="unfinished"></translation>
+        <translation>قارشى كونترول قىلىش (Ctrl+H قوزغىتىش، Ctrl+G بىكار قىلىش)</translation>
     </message>
     <message>
         <location filename="../pcscreenmanage/suspendtabbar.cpp" line="177"/>
         <location filename="../pcscreenmanage/suspendtabbar.cpp" line="214"/>
         <source>End screen projection</source>
-        <translation type="unfinished"></translation>
+        <translation>ئېكراننى پىرويېكسىيەلەشنى ئاخىرلاشتۇرۇش</translation>
     </message>
     <message>
         <location filename="../pcscreenmanage/suspendtabbar.cpp" line="213"/>
         <source>End Counter Control (Ctrl+G)</source>
-        <translation type="unfinished"></translation>
+        <translation>قارشى كونتىرول قىلىشنى ئاخىرلاشتۇرۇش (Ctrl+G)</translation>
     </message>
 </context>
 <context>
@@ -749,51 +749,51 @@
     <message>
         <location filename="../ui/view/titlebar.cpp" line="203"/>
         <source>Menu</source>
-        <translation type="unfinished"></translation>
+        <translation>تىزىملىك</translation>
     </message>
     <message>
         <location filename="../ui/view/titlebar.cpp" line="45"/>
         <source>kylin-connectivity</source>
-        <translation type="unfinished"></translation>
+        <translation>كۆپ تېرمىنالنى بىرلەشتۈرۈش</translation>
     </message>
     <message>
         <location filename="../ui/view/titlebar.cpp" line="57"/>
         <source>Minimize</source>
-        <translation type="unfinished"></translation>
+        <translation>كىچىكلىتىش</translation>
     </message>
     <message>
         <location filename="../ui/view/titlebar.cpp" line="67"/>
         <location filename="../ui/view/titlebar.cpp" line="124"/>
         <source>Maximize</source>
-        <translation type="unfinished"></translation>
+        <translation>چوڭايتىش</translation>
     </message>
     <message>
         <location filename="../ui/view/titlebar.cpp" line="77"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>تاقاش</translation>
     </message>
     <message>
         <location filename="../ui/view/titlebar.cpp" line="128"/>
         <source>Reduction</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەسلىگە قايتۇرۇش</translation>
     </message>
     <message>
         <location filename="../ui/view/titlebar.cpp" line="220"/>
         <location filename="../ui/view/titlebar.cpp" line="260"/>
         <source>Help</source>
-        <translation type="unfinished"></translation>
+        <translation>ياردەم</translation>
     </message>
     <message>
         <location filename="../ui/view/titlebar.cpp" line="221"/>
         <location filename="../ui/view/titlebar.cpp" line="262"/>
         <source>About</source>
-        <translation type="unfinished"></translation>
+        <translation>ھەققىدە</translation>
     </message>
     <message>
         <location filename="../ui/view/titlebar.cpp" line="222"/>
         <location filename="../ui/view/titlebar.cpp" line="264"/>
         <source>Quit</source>
-        <translation type="unfinished"></translation>
+        <translation>چېكىنىش</translation>
     </message>
 </context>
 <context>
@@ -801,7 +801,7 @@
     <message>
         <location filename="../ui/basewidget/transmissiondialog.cpp" line="16"/>
         <source>Current progress</source>
-        <translation type="unfinished"></translation>
+        <translation>ھازىرقى يۈرۈشۈش ئەھۋالى</translation>
     </message>
 </context>
 <context>
@@ -809,12 +809,12 @@
     <message>
         <location filename="../ui/connectinterface/usbconnectwin.cpp" line="71"/>
         <source>Connect now</source>
-        <translation type="unfinished"></translation>
+        <translation>تەڭشەش تامالاندى، دەرھال ئۇلاش</translation>
     </message>
     <message>
         <location filename="../ui/connectinterface/usbconnectwin.cpp" line="116"/>
         <source>No tutorial</source>
-        <translation type="unfinished"></translation>
+        <translation>دەرسلىك يوق</translation>
     </message>
 </context>
 <context>
@@ -822,7 +822,7 @@
     <message>
         <location filename="../projection/device/deviceui/videoform.cpp" line="776"/>
         <source>Control device not supported</source>
-        <translation type="unfinished"></translation>
+        <translation>ھازىرقى ئۇلىنىش ئۈسكىنىنى كونترول قىلىشنى قوللىمايدۇ</translation>
     </message>
 </context>
 <context>
@@ -830,54 +830,54 @@
     <message>
         <location filename="../projection/uibase/videotitle.cpp" line="66"/>
         <source>kylin-connectivity</source>
-        <translation type="unfinished"></translation>
+        <translation>كۆپ تېرمىنالنى بىرلەشتۈرۈش</translation>
     </message>
     <message>
         <location filename="../projection/uibase/videotitle.cpp" line="145"/>
         <location filename="../projection/uibase/videotitle.cpp" line="183"/>
         <source>Hide Navigation Button</source>
-        <translation type="unfinished"></translation>
+        <translation>يول باشلاش ئىستونىنى يوشۇرۇش</translation>
     </message>
     <message>
         <location filename="../projection/uibase/videotitle.cpp" line="146"/>
         <source>Stay on top</source>
-        <translation type="unfinished"></translation>
+        <translation>چوققىلاش</translation>
     </message>
     <message>
         <location filename="../projection/uibase/videotitle.cpp" line="147"/>
         <location filename="../projection/uibase/videotitle.cpp" line="209"/>
         <source>FullScreen</source>
-        <translation type="unfinished"></translation>
+        <translation>پۈتۈن ئېكران</translation>
     </message>
     <message>
         <location filename="../projection/uibase/videotitle.cpp" line="148"/>
         <source>Mouse sensitivity</source>
-        <translation type="unfinished"></translation>
+        <translation>مائۇسنىڭ سەزگۈرلۈكى</translation>
     </message>
     <message>
         <location filename="../projection/uibase/videotitle.cpp" line="149"/>
         <source>Quit</source>
-        <translation type="unfinished"></translation>
+        <translation>چېكىنىش</translation>
     </message>
     <message>
         <location filename="../projection/uibase/videotitle.cpp" line="180"/>
         <source>Show Navigation Button</source>
-        <translation type="unfinished"></translation>
+        <translation>يول باشلاش ئىستونىنى كۆرسىتىش</translation>
     </message>
     <message>
         <location filename="../projection/uibase/videotitle.cpp" line="193"/>
         <source>Stay On Top</source>
-        <translation type="unfinished"></translation>
+        <translation>چوققىلاش</translation>
     </message>
     <message>
         <location filename="../projection/uibase/videotitle.cpp" line="197"/>
         <source>Cancel Stay On Top</source>
-        <translation type="unfinished"></translation>
+        <translation>چوققىلاشنى بىكار قىلىش</translation>
     </message>
     <message>
         <location filename="../projection/uibase/videotitle.cpp" line="213"/>
         <source>Cancel FullScreen</source>
-        <translation type="unfinished"></translation>
+        <translation>پۈتۈن ئېكراننى بىكار قىلىش</translation>
     </message>
 </context>
 <context>
@@ -885,7 +885,7 @@
     <message>
         <location filename="../projection/device/deviceui/videoform.ui" line="17"/>
         <source>kylin-connectivity</source>
-        <translation type="unfinished"></translation>
+        <translation>كۆپ تېرمىنالنى بىرلەشتۈرۈش</translation>
     </message>
 </context>
 </TS>
